spec.bs

```diff
@@ -227,6 +227,32 @@ spec: html; urlPrefix: https://www.rfc-editor.org/rfc/
         1. [=set/Append=] |callerOrigin| to |topicsHistoryEntry|'s [=topics caller origins=].
   </div>
 
+  <h2 id="derive-top-5-topics-header">Derive top 5 topics</h2>
+
+  Given a [=list=] of [=topics history entries=] <var ignore=''>historyEntriesForUserTopics</var>, the browser should provide an algorithm to <dfn>derive top 5 topics</dfn>, that are believed to be valuable for the Topics callers.
```

Comment:
You should state the algorithm's expected return type. I think it's a list of 5 topic IDs?